[PATCH] Do not modify environment

WSL1 has issues when setting the path to fully qualified POSIX paths.
It ignores the other components for some reason.

---

--- ltmain.sh.org	2022-04-01 02:00:42.719606700 +0300
+++ ltmain.sh	2022-04-01 20:15:05.183466800 +0300
@@ -6324,6 +6333,8 @@ nonempty (const char *s)
 void
 lt_setenv (const char *name, const char *value)
 {
+  if (getenv("LT_SKIP_ENV") != NULL)
+    return;
   lt_debugprintf (__FILE__, __LINE__,
 		  "(lt_setenv) setting '%s' to '%s'\n",
                   nonnull (name), nonnull (value));
